security/gnupg-pkcs11-scd/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    gnupg-pkcs11-scd
+github.setup            alonbl ${name} 0.10.0 ${name}-
+
+categories              security
+license                 BSD
+maintainers             {@sstallion gmail.com:sstallion} openmaintainer
+
+description             GnuPG PKCS#11 Smart-Card Daemon
+long_description        gnupg-pkcs11 is a project to implement a BSD-licensed           \
+                        smart-card daemon to enable the use of PKCS#11 tokens with      \
+                        GnuPG
+
+homepage                https://github.com/alonbl/gnupg-pkcs11-scd
```

Comment:
this is set as default from the `github.setup` line so please remove

Reply:
Done!